lib/matplotlib/cbook.py

```diff
@@ -2692,9 +2694,12 @@ def __init__(self, function, inverse, bounded_0_1=True, check_params=None):
         elif callable(check_params):
             self._check_params = check_params
         else:
-            raise ValueError("Check params must be a callable, returning "
-                             "a boolean with the validity of the passed "
-                             "parameters, or None.")
+            raise ValueError("Check params must be a callable taking a list "
```

Comment:
thinking "Invalid check_params function" might be better as it should hopefully just force people to look at the `check_params` documentation. Partially my fault, but this is getting massively convluted and I think the error message format doesn't really lend itself to clarity.